src/main/resources/swagger/rawls.yaml

```diff
@@ -1738,6 +1761,23 @@ definitions:
         $ref: '#/definitions/MethodConfigurationName'
         description: The destination for a copied method configuration
 
+  MethodRepoConfigurationExport:
+    description: ''
```

Comment:
none of the others have a description, I was only trying to maintain consistency